### ISSUE TITLE
Global package bins don't work on Node.js v10.3.0 

### ISSUE THREAD

**Issue by rstacruz** [asdf-vm/asdf-nodejs#80]
When running bins from global packages in the latest Node.js version, you get the error:

> No such command in 10.3.0 of nodejs

## Steps to replicate

```bash
# Use the latest node.js
asdf install nodejs 10.3.0
asdf global nodejs 10.3.0

# Install a package
npm install -g gh

# Generate shims
asdf reshim nodejs

# Try to run it
gh
# No such command in 10.3.0 of nodejs
```

## Workaround

The shims are made as `.npm/bin/xxx`, but are now in `bin/xxx`. To remedy this, you can make .npm/bin available again:

```bash
ln -nfs ../bin ~/.asdf/installs/nodejs/10.3.0/.npm/bin
```

**Comment by vic** [comment]
Could probably be already fixed when using the new shim executor on asdf 0.6.4-dev (https://github.com/asdf-vm/asdf/commit/cc7886309271b21a7da3fccf8efc892b546ffbbb). Could anyone test and report back?

**Comment by sevillaarvin** [comment]
Hi, I experienced the same issue with asdf@0.8.0.

Installed asdf via
```sh
git clone https://github.com/asdf-vm/asdf.git ~/.asdf --branch v0.8.0
vim ~/.bashrc
# https://asdf-vm.com/#/core-manage-asdf
. $HOME/.asdf/asdf.sh
. $HOME/.asdf/completions/asdf.bash
```

Installed node via:
```sh
asdf install nodejs 15.2.0
asdf global nodejs 15.2.0
```

Installed npm via:
```sh
npm i -g pnpm
```

However, running `pnpm` returns `Command 'pnpm' not found`.

I confirmed that pnpm exists, but somehow not included in PATH.
```sh
ls ~/.asdf/installs/nodejs/15.2.0/.npm/bin/
pnpm  pnpx
```

I executed `asdf reshim nodejs` and now it works.

**Comment by desmap** [comment]
fwiw, have same global pnpm issue as pre-poster and could resolve with reshim; I breifly skimmed you shim docs but couldn't grok why we need this reshim extra step; there must be reason and if I spent more than 30sec then I get it Ibut I find it doc- and UI-wise not really intuitive...

**Comment by augustobmoura** [comment]
I can't reproduce it anymore. It should have been fixed since #232. If the error still occurs, please comment in this thread so we can reopen it or create a new one.
